spec/factories/return_request_notifications.rb

```diff
@@ -4,10 +4,12 @@
     date { Time.zone.now }
     item { FactoryBot.build(:pending, owner: user.id) }
     borrower { FactoryBot.build(:max) }
+    sequence(:active) { |n| n % 2 }
   end
 
   factory :invalid_return_request_notification, class: 'ReturnRequestNotification' do
     item { FactoryBot.build(:pending) }
     borrower { FactoryBot.build(:max) }
+    sequence(:active) { |n| n % 2 }
```

Comment:
Could this maybe be solved in pure CSS? 

See https://www.w3schools.com/howto/howto_css_table_zebra.asp

Reply:
Im not sure what you mean by this. The lines you have quoted are from a factory bot, that does not really use any css at all.
We also only made changes to the factories in the first place, because we somehow had failing tests after merging dev into our branch, because the factories didn't set some attributes.

Reply:
Ah, nevermind. I thought that this was some display code, but this is testing stuff.